### ISSUE TITLE
Secondary buttons don't have enough contrast

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#366]
Our secondary buttons don't have enough contrast, the red has to be made darker.

> 
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.82:1. Recommendation: change background to #d83d2d.

![screen shot 2016-06-16 at 2 05 22 pm](https://cloud.githubusercontent.com/assets/1701077/16133168/50e60c1c-33cb-11e6-8f66-6c2f37093ed5.png)

**Comment by msecret** [comment]
Merged into #374
